GoogleForCreators/web-stories-wp
# FTUE: Find missing 'done' svg

## Context

Done checkmark was missing in the FTUE help center.

## Summary

Updates the colors for the paths in the svg.

## Relevant Technical Choices

Importing the svg changes all the `stroke` and `fill` attributes to `currentColor`, so each part of the svg was inheriting the same `color`.
<img width="280" alt="Screen Shot 2021-03-19 at 1 32 46 PM" src="https://user-images.githubusercontent.com/22185279/111833421-c165d080-88b7-11eb-9d72-9e8f42ed2996.png">

I couldn't figure out how to import the svg without the svg having its `stroke` and `fill` changed to `currentColor`, so I decided to style it directly through the styled component.

If somebody knows how to change this lmk!

## To-do

none

## User-facing changes

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/111833172-6e8c1900-88b7-11eb-98df-3036ac81d60e.png)|![image](https://user-images.githubusercontent.com/22185279/111833139-5e743980-88b7-11eb-9e1e-dc1783998eff.png)|


## Testing Instructions

1. go to editor
2. open help center for ftue
3. click the last tip, then click through until you get to the end. Should see the svg

### QA






This PR can be tested by following these steps:


1. go to editor
2. open help center for ftue
3. click the last tip, then click through until you get to the end. Should see the svg

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6850
